File: 30 Days of Code/Python/day_7.py
"""
Created on Wed Jul  26 15:45:25 2017

@author: Dmitry White
"""

# TODO: Given an array, A, of N integers, print A's elements in reverse order
# as a single line of space-separated numbers.

n = int(input())
arr = reversed(input().split())
print(" ".join(arr))

File: 30 Days of Code/JavaScript/day_7.js
/*
  Created on Sat May 13 14:29 2023

  @author: Dmitry White
*/

/*
  TODO: Given an array, A, of N integers, print A's elements in reverse order
  as a single line of space-separated numbers.
*/

const reverseOutput = (arr) => {
  console.log(arr.reverse().join(' '));
};

reverseOutput([1, 4, 3, 2]);
